app/views/cats/index.html.haml

```diff
@@ -0,0 +1,5 @@
+%h1 Welcome to Cattastic!
+- @cats.each do |cat|
+    .cat
+    = image_tag cat['url']
```

Comment:
Mind the indentation of this line if you want the img tag to be wrapped in `.cat` div. Check the rendered source in your browser.

Reply:
makes sense - have changed